loadbalancer.tf

```diff
@@ -65,7 +65,7 @@ resource "volterra_http_loadbalancer" "this" {
   disable_rate_limit              = true
   round_robin                     = true
   service_policies_from_namespace = true
-  no_challenge                    = false
+  no_challenge                    = var.disable_js_challenge
```

Comment:
This is not the right way to do it, you can use dynamic block in terraform to disable it or you can configure policy challenge and manage it that way